src/components/general/Blog.vue

```diff
@@ -9,12 +25,122 @@ export default {
 	props: {
 		limit: Number
 	},
+	data() {
+		return {
+			blogData: []
+		}
+	},
+	methods: {
+		toText(htmlText) {
+			let plainText = document.createElement('div')
+			plainText.innerHTML = htmlText
+			htmlText = plainText.innerText
+			return htmlText
+		}
+	},
+	beforeMount() {
+		let referenceThis = this
+		let httpRequest = new XMLHttpRequest();
+		httpRequest.onreadystatechange = function() {
+			if (this.readyState == 4 && this.status == 200) {
+				referenceThis.blogData = JSON.parse(this.responseText);
+			}
+		};
+		httpRequest.open("GET", "http://test.cutc.ca/mediumPosts.php", true);
+		httpRequest.send();
+	}
 }
 </script>
 
 <style scoped>
+#blog {
+	max-width: 1120px;
+}
+
 a {
 	color: var(--main-color);
+}
+
+router-link {
 	text-decoration: underline;
 }
```

Comment:
The underline isn't showing up on my screen, and I believe it should (to match the speakers section).

![image](https://user-images.githubusercontent.com/41529410/117393705-b5af8700-aec2-11eb-8790-664c03b9c8b2.png)